src/ondemand.coffee

```diff
@@ -231,119 +86,185 @@ module.exports = (robot) ->
+            message =
+              if data.length then (invasion.toString() for invasion in data).join('\n\n')
+              else "#{md.codeMulti}Operator, there are no invasions at the moment#{md.blockEnd}"
+            res.send message  
+  robot.respond /news/, (res) ->
+    userDB.getPlatform res.message.room, (err, platform) ->
+      if err
+        return robot.logger.error err
+      checkWorldstate platform
+      setTimeout (-> res.send worldStates[platform].getNewsString()), 300
+      
+  robot.respond /primeaccess/, (res) ->
+    userDB.getPlatform res.message.room, (err, platform) ->
+      if err
+        return robot.logger.error err
+      checkWorldstate platform
+      setTimeout (-> res.send worldStates[platform].getPrimeAccessString()), 300
+  
+  robot.respond /update/, (res) ->
+    userDB.getPlatform res.message.room, (err, platform) ->
+      if err
+        return robot.logger.error err
+      checkWorldstate platform
+      setTimeout (-> res.send worldStates[platform].getUpdatesString()), 300
+
   robot.respond /rewards/, (res) ->
-    res.send util.format('%sMission rewards%shttp://rewards.morningstar-wf.com%s',
-                           dsUtil.linkBegin, dsUtil.linkMid, dsUtil.linkEnd)
+    res.send "#{md.codeMulti}#{md.linkBegin}Mission rewards #{md.linkMid} http://rewards.morningstar-wf.com#{md.linkEnd}#{md.blockEnd}"
+    
+  robot.respond /shield(?:\s+([\d\s]+))?/, (res) ->
+    pattern3Params = new RegExp(/^(\d+)(?:\s+(\d+)\s+(\d+))?$/)
+    robot.logger.debug util.format('matched shield command. matching string: %s', res.match[1])
+    params = res.match[1]
+    
+    if pattern3Params.test(params)
+      shields = params.match(pattern3Params)[1]
+      baseLevel = params.match(pattern3Params)[2]
+      currentLevel = params.match(pattern3Params)[3]
+      
+      robot.logger.debug 'Entered 3-param shield'
+      shieldString = dsUtil.shieldString dsUtil.shieldCalc(shields, baseLevel, currentLevel), currentLevel    
+    else
+      robot.logger.debug 'Entered 0-param shield'
+      shieldInstruct3 = 'shield (Base Shelds) (Base Level) (Current Level) calculate shields and stats.'
+      shieldString = "#{md.codeMulti}Possible uses include:#{md.lineEnd}#{shieldInstruct3}#{md.blockEnd}"
+    res.send shieldString
+  robot.respond /simaris/, (res) ->
+    res.send "#{md.codeMulti}No info about Synthesis Targets, Simaris has left us alone#{md.blockEnd}"
+    
+  robot.respond /sortie/, (res) ->
+    userDB.getPlatform res.message.room, (err, platform) ->
+      if err
+        return robot.logger.error err
+      checkWorldstate platform
+      setTimeout (-> res.send worldStates[platform].getSortieString()), 300
+        
+  robot.respond /tutorial(.+)/, (res) ->
+    tutorial = res.match[1]
+    focusReg = /\sfocus/
+    if focusReg.test tutorial
+      res.send "#{md.codeMulti}#{md.linkBegin}Warframe Focus #{md.linkMid} https://www.youtube.com/watch?v=IMltFZ97oXc #{md.linkEnd}#{md.blockEnd}"
```

Comment:
Same as above, additional spaces break markdown formatting. Should be `...Focus#{md.linkMid}https://...`